scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
-                Number of random initializations of the starting
-                permutation matrix that the FAQ algorithm will undergo.
+        A dictionary of solver options.
+
+            init_J : 2d-array or "barycenter" (default = "barycenter")
+                The initial (guess) permutation matrix or search "position"
+                :math:`J`.
+
+                :math:`J` need not be a proper permutation matrix;
+                however, it must have the same shape as `cost_matrix` and
+                `dist_matrix`, and it must be doubly stochastic: each of its
+                rows and columns must sum to 1.
+
+                If unspecified or ``"barycenter"``, the non-informative "flat
+                doubly stochastic matrix" :math:`1*1^T/n`, where :math:`n`
+                is the number of nodes and :math:`1` is a :math:`n \times 1`
+                array of ones, is used. This is the "barycenter" of the
+                search space of doubly-stochastic matrices.
+            init_weight : float in range [0, 1] (default = 1)
+                Allows the user to specify the weight of the provided
+                search position :math:`J` relative to random perturbations
+                :math:`K`.
+
+                The algorithm will be repeated :math:`k` times
+                from randomized initial search positions
+                :math:`P_0 = (\alpha J + (1- \alpha) K`, where
+                :math:`J` is given by option `init_J`,
+                :math:`\alpha` is given by option `init_weight`,
+                :math:`k` is given by option `init_k`, and
+                :math:`K` is a random doubly stochastic matrix.
+            init_k : int, positive (default = 1)
+                The number of randomized initial search positions :math:`P_0`
+                from which the FAQ algorithm will proceed.
             maxiter : int, positive (default = 30)
-                Integer specifying the max number of Franke-Wolfe iterations.
-                FAQ typically converges with modest number of iterations.
+                Integer specifying the max number of Franke-Wolfe iterations
+                per initial search position. FAQ typically converges within a
+                modest number of iterations.
             shuffle_input : bool (default = True)
                 To avoid artificially high or low matching due to inherent
                 sorting of input adjacency matrices, gives users the option
-                to shuffle the nodes of `cost_matrix`. Results are then
-                unshuffled so that returned `col_ind` matches the node order
-                of inputs.
+                to shuffle the nodes. Results are then unshuffled so that the
+                returned results correspond with the node order of inputs.
             eps : float (default = 0.05)
-                A positive, threshold stopping criteria such that Franke-
-                Wolfe continues to iterate while Frobenius norm of
```

Comment:
I thought if we're calling it a stopping criterion, we should phrase this in terms of when iteration stops instead of when it continues.